rest/admin-api/src/main/java/org/keycloak/admin/api/client/ClientApiSpi.java

```diff
@@ -0,0 +1,35 @@
+package org.keycloak.admin.api.client;
+
+import org.keycloak.common.Profile;
+import org.keycloak.provider.Provider;
+import org.keycloak.provider.ProviderFactory;
+import org.keycloak.provider.Spi;
+
+public class ClientApiSpi implements Spi {
```

Comment:
Why is there a ClientApiSpi?

Shouldn't it just be AdminApi or something? and allow overwriting what handles the `/client` API?